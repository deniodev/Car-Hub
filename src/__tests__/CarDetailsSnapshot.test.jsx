// import { render } from '@testing-library/react';
// import { MemoryRouter, Routes, Route } from 'react-router-dom';
// import { Provider } from 'react-redux';
// import configureStore from 'redux-mock-store';
// import { describe, test, expect } from 'vitest'; // Importing from vitest
// import CarDetails from '../../src/components/Main/CarDetails';

// const mockStore = configureStore([]);

// describe('CarDetails', () => {
//     test('matches snapshot', () => {
//       const carData = {
//         id: 47,
//         name: 'Honda Civic',
//         image: 'linktoImage.png',
//         description: "Expedita quia dolore",
//         price: "56413.84",
//       };
  
//       const initialState = {
//         cars: {
//           cars: [carData],
//         },
//       };
  
//       const store = mockStore(initialState);
  
//       const { container } = render(
//         <Provider store={store}>
//           <MemoryRouter initialEntries={['/car/1']}>
//             <Routes>
//               <Route path="/cars/:id" element={<CarDetails />} />
//             </Routes>
//           </MemoryRouter>
//         </Provider>
//       );
  
//       expect(container).toMatchSnapshot();
//     });
//   });