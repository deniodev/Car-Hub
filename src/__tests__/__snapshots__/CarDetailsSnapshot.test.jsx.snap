// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`CarDetails > matches snapshot 1`] = `<div />`;
